app/models/game.rb

```diff
@@ -9,4 +9,6 @@ class Game < ApplicationRecord
   has_many :comments
   has_many :pieces
 
+  scope :available, -> { where(creating_user_id: nil).or.where(invited_user_id: nil)}
```

Comment:
I am not sure that we would need a check for `creating_user_id` since a game cannot exist without a `creating_user_id` (see Game.rb). Could you explain why you chose this?

Reply:
Also, it looks like your Game.rb doesn't match the master file even though Git is not saying so... which is concerning. Master currently has:

```
  class Game < ApplicationRecord
  belongs_to :creating_user, :class_name => 'User', :foreign_key => 'creating_user_id'
  belongs_to :invited_user, :class_name => 'User', :foreign_key => 'invited_user_id', optional: true
  belongs_to :first_move, :class_name => 'User', :foreign_key => 'first_move_id', optional: true
  belongs_to :winner, :class_name => 'User', :foreign_key => 'winner_id', optional: true
  has_many :moves
  has_many :comments
  has_many :pieces

end
```

Reply:
Shane, my logic was, the scope is allowing a user to join/start a game, so that person has to be either the creating user or the invited user. BUt now that you've challenged the thought maybe I read it wrong and will have to go back to the drawing board.

Reply:
as for your other comment not sure as I just checked as well and its saying my master is up to date with origin. hmmm

Reply:
Sometimes git will say it is up to date (at least in my experience), but you still need to run `git pull` in order to pull in most recent merges to the master branch from the repo. 

I'm thinking that we just need the scope to exclude the or statement like so:
`scope :available, -> { where(invited_user_id: nil) }`